src/styles/reset.js

```diff
@@ -3,6 +3,11 @@
 import { injectGlobal } from 'styled-components';
 
 injectGlobal`
+
+html {
+  background-color: #d1f4ff;
```

Comment:
I don't know if it's just me but this colour kind of hurts my eyes :see_no_evil:

Reply:
@matthewdking 
Its more of an opinion really :stuck_out_tongue_winking_eye: 
One of the main thing we do know from our customer is that she like the design
i dont think we can really change the look of what we already presented to her :sweat_smile: